projects/dashboards/src/widgets/conditional/conditional-widget-renderer.component.ts

```diff
@@ -8,23 +8,15 @@ import { ConditionalModel } from './conditional.model';
 @Renderer({ modelClass: ConditionalModel })
 @Component({
   selector: 'ht-conditional-widget',
-  styleUrls: ['./conditional-widget-renderer.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
-  template: `
-    <div class="conditional-widget">
-      <ng-container [hdaDashboardModel]="this.getModel() | async"> </ng-container>
-    </div>
-  `
+  template: ` <ng-container [hdaDashboardModel]="this.getChildModel() | async"> </ng-container> `
```

Comment:
by virtue of making this our fetch data implementation, this is already available as `this.data$` - we could inline line 16 into fetchData.

Reply:
I'll fix this locally. Thanks.